src/SimplePDPService.sol

```diff
@@ -112,9 +115,10 @@ contract SimplePDPService is PDPListener, PDPRecordKeeper, Initializable, UUPSUp
     }
 
     // The start of the challenge window for the current proving period
+    // If proof set state not yet initialized assume proving period starts now
```

Comment:
Why ? Can you explain what goes wrong if we don't do this ?

Reply:
I can't get the info I need as SP software consuming this to start up my proofset.  Would you prefer putting this in a separate function?

Reply:
Moving this functionality to `initChallengeWindowStart`

Reply:
Done